### PR TITLE
Add Bigtable API

### PR DESCRIPTION
**Pull request by evildour**
For some reason VS saw fit to remove some commas from apis.json as well.

### CODE REVIEW

File: generateapis.sh [hunk 1]
```diff
@@ -153,6 +153,8 @@ generate_api Google.Cloud.Spanner.V1 google/spanner/v1 spanner.yaml
 generate_api Google.Cloud.Spanner.Admin.Database.V1 google/spanner/admin/database/v1 spanner_admin_database.yaml
 generate_api Google.Cloud.Spanner.Admin.Instance.V1 google/spanner/admin/instance/v1 spanner_admin_instance.yaml
 
+generate_api Google.Cloud.Bigtable.V2 google/bigtable/v2 bigtable.yaml
```
Comment (jskeet) [inline generateapis.sh]: Remove the blank line before this?

Comment (evildour) [inline generateapis.sh]: Done. Do you want me to remove the blank above the spanner apis as well while I'm at it?

**Comment by evildour** [comment]
No problem. I'll add them back in.

File: generateapis.sh (hunk 1 above)
Comment (jskeet) [inline generateapis.sh]: Don't bother for this PR - I'll create a new one which actually sorts them all.

**Comment by jskeet** [comment]
AppVeyor change suggests you need to rebase from master and rerun generateprojects.sh... I modified the generator earlier today, sorry!

**Comment by evildour** [comment]
No it's my fault: I actually did rebase before creating the PR - I just forgot to regenerate.

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.csproj [hunk 2]
```diff
@@ -14,12 +14,19 @@
...
     <Copyright>Copyright 2017 Google Inc.</Copyright>
     <Authors>Google Inc.</Authors>
-    <IconUrl>https://cloud.google.com/images/gcp-icon-64x64.png</IconUrl>
+    <PackageIconUrl>https://cloud.google.com/images/gcp-icon-64x64.png</PackageIconUrl>
     <PackageLicenseUrl>http://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/GoogleCloudPlatform/google-cloud-dotnet</PackageProjectUrl>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/GoogleCloudPlatform/google-cloud-dotnet</RepositoryUrl>
   </PropertyGroup>
+  <ItemGroup>
```
Comment (jskeet) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.csproj]: Interesting that this got moved. Suggests I may have a bug in my generator. Will have a look...

Comment (evildour) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.csproj]: Yeah, I noticed when I regenerated that it was in there twice, so I manually deleted one of them.

Comment (jskeet) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.csproj]: Eek. It *really* shouldn't have been in there twice. Hmm. Will definitely look now :)

Comment (evildour) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.csproj]: Oh, I think I see the issue: when the file is not there, the packingElement is placed before the dependenciesItemGroup, but when the file exists already, it goes after it.

Comment (evildour) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.csproj]: Want me to fix it? Or are you already in the process?

Comment (jskeet) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.csproj]: Nope, I'm fixing it. It's very simple.

**Pull request merged**